### DISCUSSION TITLE
Why the fit_interval_censoring returns two different survival function?

### DISCUSSION THREAD

**Discussion by eduardo-janotti** [CamDavidsonPilon/lifelines#1457] [Q&A] [answered]
Hello.

I've been trying to use fit_interval_censoring function and going through lifelines' documentation to understand the results. 

Every time that i use this, i receive two different survival functions, one called upper and the other lower, but i can't find anywhere in the documentation, or even in other places, the reason why the algorithm returns two different survival functions instead of just one, i can only find that it uses the Turnbull estimator. However, i couldn't find examples where the estimator produced two functions. Would you mind explaining why it returns two functions?

**Comment by CamDavidsonPilon** [comment, marked as answer]
This is a good question! Something that should go in the docs too. 

For Turnbull-interval estimator, there's not a _single_ optimal solution, but a range of optimal solutions: any curve that fits in between the two bounds you are provided back is optimal. 

A perhaps simpler example: given the sequence of data `0, 8, 10, 20` - what is the median? Well, any number between 8 and 10 would be a median (so really I should ask "what is _a_ median?").

**Reply by eduardo-janotti** [reply]
thanks for the prompt answer, that really helped clarify things.
